backend/de.metas.acct.base/src/main/java-legacy/org/compiere/acct/FactLineBuilder.java

```diff
@@ -214,10 +215,16 @@ else if (docLine != null && (docLine.getAmtAcctDr() != null || docLine.getAmtAcc
 			line.setAD_Org_ID(orgId.getRepoId());
 		}
 		//
+		final BPartnerLocationId bPartnerLocationId = getBPartnerLocationId();
+		if (bPartnerLocationId != null)
```

Comment:
IMHO here we shall use the same code that  currently have in FactAcctBL (and i suggested to be moved in FactLine)